src/transformers/configuration_bart.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         normalize_embedding=True,
         static_position_embeddings=False,
         add_bias_logits=False,
+        variant="bart",
```

Comment:
Not sure whether this is the cleanest way to do it.. `variant` defaults to `"bart"` , but then is later checked for `"prelayernorm"`. For me `"bart"` and `"prelayernorm"` don't really belong to the same category (one is "models" the other one would be more "model architecture")

Reply:
Maybe we can just change "bart" to "bart_layernorm" or "postlayernorm"?

Reply:
@sshleifer - what do you think?